packages/data-point/lib/entity-types/base-entity/__snapshots__/resolve.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ResolveEntity.resolveEntity if custom typeCheck throws then fail 1`] = `[Error: custom type error]`;
+
+exports[`ResolveEntity.resolveEntity inputType - throws error if inputType does not pass 1`] = `[Error: model:c.0 did not pass type check, received: 'foo' of type: string, expected: number]`;
+
+exports[`ResolveEntity.resolveEntity outputType - throws error if outputType does not pass 1`] = `[Error: model:c.1 did not pass type check, received: 1 of type: number, expected: string]`;
```

Comment:
I think we should rephrase these messages a bit. for example, `received: 1 of type: number` sounds like it received 1 value, not that it received the value 1. Adding `"the value"` after the word `"received"` might be more clear:

`model:c.1 did not pass type check, received the value 1 of type "number", but expected a value of type "string"`